calcom/cal.com
# Further sanitation for the auth callback URL. Removal of double forward slashes in the path.

## What does this PR do?
It further sanitizes the auth `callbackURL` from the changes made in #2086.
It assures only one forward slash will be used to join the base URL and the callback path  
 
Fixes #2196

## Type of change

## How should this be tested?
- Simulate a login flow with any credentials you have. Be sure to clean up the session to start with an empty callback
- Check the console before this PR and after. You will note that the next-auth error is gone.

## Checklist
- I haven't added tests that prove my fix is effective or that my feature works
